cob_bringup/CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ set(robotlist
   cob3-9
   cob4-1
   cob4-2
+  cob4-3
+  cob4-4
```

Comment:
same as above....changed my mind....it's better to fix something than to be picky about such small things....Thanks for detecting it!